design-system/materials/internals/definition.yaml

```diff
@@ -99,28 +51,75 @@ choiceGroupsByTheme:
         color-info-85: 'hsl(203.05555555555554, 93.9130434783%, 85%)'
         color-info-90: '#CEEBFD'
         color-info-95: 'hsl(203.05555555555554, 93.9130434783%, 95%)'
-        color-warning: '#f16d0e'
-        color-warning-25: 'hsl(25.110132158590307, 89.0196078431%, 25%)'
-        color-warning-40: 'hsl(25.110132158590307, 89.0196078431%, 40%)'
-        color-warning-60: 'hsl(25.110132158590307, 90%, 60%)' # This design token is not used in the default theme; it's included only for export purposes
-        color-warning-85: 'hsl(25.110132158590307, 89.0196078431%, 85%)'
-        color-warning-95: 'hsl(25.110132158590307, 89.0196078431%, 95%)'
-        color-error: '#e60050'
-        color-error-25: 'hsl(339.1304347826087, 100%, 25%)'
-        color-error-40: 'hsl(339.1304347826087, 100%, 40%)'
-        color-error-85: 'hsl(339.1304347826087, 100%, 85%)'
-        color-error-95: 'hsl(339.1304347826087, 100%, 95%)'
         color-solid: '#1a1a1a'
         color-solid-02: 'hsl(0deg 0% 10% / 2%)'
         color-solid-05: 'hsl(0deg 0% 10% / 5%)'
         color-solid-10: 'hsl(0deg 0% 10% / 10%)'
         color-surface: '#fff'
         color-transparent: 'transparent'
-        color-success: 'hsl(152, 77%, 39%)' # Design token copied from recolouring theme
-        color-success-25: 'hsl(155, 84%, 20%)' # Design token copied from recolouring theme
-        color-success-40: 'hsl(155, 90%, 24%)' # Design token copied from recolouring theme
-        color-success-85: 'hsl(144, 69%, 80%)' # Design token copied from recolouring theme
-        color-success-95: 'hsl(141, 76%, 92%)' # Design token copied from recolouring theme
+        color-primary: 'hsl(240, 64%, 58%)'
```

Comment:
Any reason to move this block of tokens from where they were? 🤔

Reply:
Oh wait, we are still leaving the design tokens for the recolouring block of the `choiceGroupsByTheme` as it is right  ?

Reply:
I'm just saying that the colors tokens started with the `color-primary` and now they start with the `color-neutral`.
I was expecting to see just the values of the tokens changed, not the tokens moved around.

If it's not clear, I can push a commit to explain what I mean if that's ok for you.